Eastern-Research-Group/mywaterway
# Feature/280 cyan integration

## Related Issues:
* [HMW-280](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-280)
* [HMW-370](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-370)
* [HMW-371](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-371)

## Main Changes:
* Added a CyAN layer to the map on the **Community** page.
    * This is currently only listed on the **Monitoring** tab.
* Added an accordion/popup panel for the CyAN waterbodies, with daily statistics, a bar chart showing a weekly summary,  and a slider for selecting the day to display.
* Renamed instances of "Current Water Conditions" to "USGS Sensors".
* Turned off initial "Past Water Conditions" visibility on the **Monitoring** tab.

## Steps To Test:
1. Navigate to a location with a CyAN waterbody, e.g. [Lake Okeechobee](http://localhost:3000/community/030902010200/monitoring).
2. Confirm that the "CyAN Waterbodies" layer is visible, and that the "Past Water Conditions" layer is not visible.
3. Confirm that the counts match the number of list items, and that the visibility toggles work as expected.
4. Open the accordion item for _Lake Okeechobee_.
5. Click the hamburger icon in the bar chart, select "Download PNG image", and verify the downloaded image has labels added.
6. Verify the date slider ends with the previous day, and that the date initially selected has data.
7. Adjust the slider handle, and verify the daily statistics update, as does the satellite image placed on the map.
